public/app/plugins/datasource/influxdb/components/__snapshots__/ConfigEditor.test.tsx.snap

```diff
@@ -81,7 +81,7 @@ exports[`Render should disable basic auth password input 1`] = `
     }
     defaultUrl="http://localhost:8086"
     onChange={[MockFunction]}
-    showAccessOptions={true}
+    showAccessOptions={false}
```

Comment:
can we set the default value for the `showAccessOptions` prop be false if undefined so we can remove these?

Reply:
I missed that these were snapshot files that are used in unit tests, thanks Beto!